lerna.json

```diff
@@ -0,0 +1,23 @@
+{
+  "packages": [
+    "packages/*",
+    "packages/ngx-conveyer/projects/ngx-conveyer"
```

Comment:
here is some other component code.

Reply:
That's fixed in the later commit
- https://github.com/WoodNeck/egjs-view3d/blob/feat-cfc/lerna.json